### PR TITLE
Review B3 - sky dragons

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json [hunk 1]
```diff
@@ -1248,7 +1176,60 @@
...
+                            ]
+                        },
+                        "selector": "jaws-damage"
+                    },
+                    {
+                        "damageType": "positive",
+                        "diceNumber": 1,
+                        "dieSize": "d12",
+                        "key": "DamageDice",
+                        "label": "PF2E.NPCAbility.SkyDragon.DivineLighningPositive",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json:1220]:
```suggestion
                        "label": "PF2E.NPCAbility.SkyDragon.DivineLightningPositive",
```

File: packs/data/pathfinder-bestiary-3.db/adult-sky-dragon.json [hunk 2]
```diff
@@ -1248,7 +1283,60 @@
...
+                            ]
+                        },
+                        "selector": "jaws-damage"
+                    },
+                    {
+                        "damageType": "positive",
+                        "diceNumber": 1,
+                        "dieSize": "d12",
+                        "key": "DamageDice",
+                        "label": "PF2E.NPCAbility.SkyDragon.DivineLighningPositive",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/adult-sky-dragon.json:1327]:
```suggestion
                        "label": "PF2E.NPCAbility.SkyDragon.DivineLightningPositive",
```

File: packs/data/pathfinder-bestiary-3.db/adult-sky-dragon.json [hunk 3]
```diff
@@ -1248,7 +1283,60 @@
...
+                        "priority": 50,
+                        "toggleable": true,
+                        "value": false
+                    },
+                    {
+                        "damageType": "good",
+                        "diceNumber": 1,
+                        "dieSize": "d12",
+                        "key": "DamageDice",
+                        "label": "PF2E.NPCAbility.SkyDragon.DivineLighningGood",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/adult-sky-dragon.json:1310]:
```suggestion
                        "label": "PF2E.NPCAbility.SkyDragon.DivineLightningGood",
```

File: packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json [hunk 4]
```diff
@@ -1248,7 +1176,60 @@
...
+                        "priority": 50,
+                        "toggleable": true,
+                        "value": false
+                    },
+                    {
+                        "damageType": "good",
+                        "diceNumber": 1,
+                        "dieSize": "d12",
+                        "key": "DamageDice",
+                        "label": "PF2E.NPCAbility.SkyDragon.DivineLighningGood",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json:1203]:
```suggestion
                        "label": "PF2E.NPCAbility.SkyDragon.DivineLightningGood",
```

File: packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json (hunk 1 above)
Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/ancient-sky-dragon.json]: Oops - thanks for finding this :-)

**Pull request merged**